vividus-plugin-rest-api/src/main/java/org/vividus/archive/steps/ArchiveSteps.java

```diff
@@ -107,21 +113,37 @@ public void verifyArchiveContainsEntries(DataWrapper archiveData, List<NamedEntr
     {
         Set<String> entryNames = ZipUtils.readZipEntryNamesFromBytes(archiveData.getBytes());
 
+        if (entryNames.isEmpty())
+        {
+        	softAssert.recordFailedAssertion("There are no files in the archive");
+        	return;
+        }
```

Comment:
even if the archive is empty we still should do verifications since there might me known issues attached to it https://docs.vividus.dev/vividus/latest/tests-configuration.html#_known_issues

Reply:
done